js/header.js

```diff
@@ -138,7 +139,7 @@
             navInfo.primary.toggle.addEventListener('click', handlePrimaryMenuToggleClick);
           }
         } else {
-          if (navInfo.primary.toggle) {
+          if (Object.keys(navInfo).includes('primary') && navInfo.primary.toggle) {
```

Comment:
```suggestion
          if (navInfo.primary?.toggle) {
```

It's also good as-is. This just has the same effect while being a bit more readable.

Reply:
Just ran across this tab again: I didn't spot it at first, but even without resorting to `primary?`, it's still possible to simplify `Object.keys()` plus `includes()`. The js object prototype has a method specifically for this, `hasOwnProperty()`:

```suggestion
          if (navInfo.hasOwnProperty('primary') && navInfo.primary.toggle) {
```